huaweicloud/services/acceptance/dew/resource_source_huaweicloud_kms_key_material_test.go

```diff
@@ -0,0 +1,136 @@
+package dew
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/kms/v1/keys"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/dew"
+)
+
+func getKmsKeyMaterialResourceFunc(conf *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	client, err := conf.KmsKeyV1Client(acceptance.HW_REGION_NAME)
+	if err != nil {
+		return nil, fmt.Errorf("error creating kms client: %s", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("error creating KMS client: %s", err)
```
`KMS` has a specific meaning and needs to be capitalized. Please check if there are similar questions elsewhere.